lib/parser.rb

```diff
@@ -1,42 +0,0 @@
-require "treetop"
```

Comment:
Should this file be in here (and the treetop stuff below?)

Reply:
A rubocop issue that took me way to long to find is that if you have a class named "Parser" it croaks with a `uninitialized constant RuboCop::AST::Parser (NameError)` I split up the PRs because the other one is the meat of it, and this is to make consistent formatting.

Reply:
You wouldn't wanna do this against master, and then just add the new stuff properly formatted in the new PR?